### ISSUE TITLE
anking cloze reveal one by one card

### ISSUE THREAD

**Issue by ankibhavesh** [AnKingMed/AnKing-Note-Types#29]
#### Problem description

*Please describe the issue concisely in here. In case of an error: Walk us through the steps you took to get there. What happened? What did you expect to happen?*

#### Checklist

*Please replace the space inside the brackets with an **x** if the following items apply:*

 - [ ] I've restarted Anki to see if it helps
 - [ ] I've verified that I use the latest version of the note type by redownloading it from AnkiWeb
 - [ ] I've tried to disable add-ons to see if there are any interactions present
 - [ ] My issue disappears when I hold shift while starting Anki.
 - [ ] I've checked if anyone else reported this problem before by looking through the issue reports. I also checked to see if there is a section about known issues in the add-on description, documentation, or README.
 

#### What Anki version are you using?

*Please open Anki, go to Help → About (windows) or Anki → About Anki (mac) to find the version. Edit the value between the backticks below

```
2.1.##
```
#### What Operating system are you using? (i.e. MacOS, Windows, Linux)

*Please include OS version as well if possible*

#### Error message (if any)

*If you've received an error message, please copy and paste it between the backticks below:*

```

```

**Comment by AnKingMed** [comment]
I'm sorry, I can't tell from your answer what the issue is

**Comment by ankibhavesh** [comment]
anking note card templet - reveal one by one note type

if their are notes in the extra section of the template and suppose i have put 3 cloze which are to be revealed one by one
but as soon as i reveal one of the  crown the extra section also pops open revealing all the notes while still 2crown are yet to be revealed

**Comment by ankibhavesh** [comment]
![image](https://user-images.githubusercontent.com/96439843/146810263-a70770b2-f7d7-4a16-aa9f-a4a3e2dc7a57.png)
extra section gets revealed while crowns are yet to be revealed

**Comment by AnKingMed** [comment]
That is the limitations of Anki, not the note. When the note is flipped over, the backside of the card will be flipped which includes the extra and any other content. You can use the Personal Notes or other fields to avoid that problem

**Comment by akavi1** [comment]
I figured out how to make the extra field hidden until all the clozes are revealed by combining bits of code I found online. Only problem is that this note type only works on the desktop Anki; I can't get it to work on iphone or ipad. 
Unfortunately, I don't know enough javascript to actually code anything but maybe someone can build on this. Would be great if it were possible to combine this with the new Anking Cloze one by one note.
[NoteTemplate.apkg.zip](https://github.com/AnKingMed/AnKing-Note-Types/files/7771911/NoteTemplate.apkg.zip)
